src/incidents/report/ReportIncident.tsx

```diff
@@ -131,6 +149,21 @@ const ReportIncident = () => {
           />
         </Column>
       </Row>
+      <Row>
+        <Column md={6}>
+          <div className="form-group patient-typeahead">
+            <Label htmlFor="patientTypeahead" text={t('incidents.reports.patient')} />
+            <Typeahead
+              id="patientTypeahead"
+              placeholder={t('incidents.reports.patient')}
```

Comment:
We need to define this key in the translations 
![image](https://user-images.githubusercontent.com/5468546/103351046-99ca4d80-4aaa-11eb-81ef-1b081ec05d6c.png)